surfinBH/_fit_evaluators/fit_7dq4Emri.py

```diff
@@ -0,0 +1,224 @@
+import numpy as np
+from surfinBH import surfinBH
+import warnings
+import sys
+
+#=============================================================================
+class Fit7dq4Emri(surfinBH.SurFinBH):
+    """ A class for the NRSur7dq4EmriRemnant model presented in Boschini et al.,
+    arxiv:[TO_BE_CHANGED] (hereafter THE PAPER). 
+    
+    This model predicts the final mass mf and final spin chif, for the remnants 
+    of precessing binary black hole systems. 
+    The fits are done using Gaussian Process Regression (GPR) and also 
+    provide an error estimate along with the fit value.
+
+    This model has been trained in the parameter space:
+        NR: q <= 4, |chiA| <= 0.8, |chiB| <= 0.8
+        EMRI: 100 <= q <= 1000 , |chiA| <= 1, |chiB| <= 1
+
+    It interpolates remnant properties at:
+        4 < q < 100
+
+    It extrapolates reasonably to:
+        q > 1000 , |chiA| <= 1, |chiB| <= 1
+
```

Comment:
Instead of lines 20 - 24, just say:
But the model can be evaluated at arbitrary mass ratios.